### PR TITLE
Basic sandbox mode implemented. Added cache for WebView.

### PR DESCRIPTION
**Pull request by manzhos-nikolai**
sandbox

### CODE REVIEW

**Comment by Gear61** [comment]
Oh wow, this is crazy. I haven't been on GitHub recently (just started working again), but I'll get to this pull request as soon as I can!

File: app/src/main/java/randomappsinc/com/sqlpractice/Activities/WebActivity.java [hunk 1]
```diff
@@ -40,7 +41,8 @@ protected void onCreate(Bundle savedInstanceState) {
             @SuppressWarnings("deprecation")
             @Override
             public void onReceivedError(WebView view, int errorCode, String description, String failingUrl) {
-                Utils.showSnackbar(parent, getString(R.string.webpage_fail));
+                //WebView displays error message by itself so it's probably not necessary do display another one, also this snackbar appears when page loaded from cache.
```
Comment (Gear61) [inline app/src/main/java/randomappsinc/com/sqlpractice/Activities/WebActivity.java]: Good point.

**Comment by manzhos-nikolai** [comment]
Ok, thanks for letting me know, if you have any other suggestions regarding sandbox mode i'll glad to assist.

File: app/src/main/java/randomappsinc/com/sqlpractice/Activities/WebActivity.java [hunk 2]
```diff
@@ -49,7 +51,7 @@ public void onReceivedError(WebView view, WebResourceRequest req, WebResourceErr
                 onReceivedError(view, rerr.getErrorCode(), rerr.getDescription().toString(), req.getUrl().toString());
             }
         });
-
+        webView.getSettings().setCacheMode(WebSettings.LOAD_DEFAULT);
```
Comment (Gear61) [inline app/src/main/java/randomappsinc/com/sqlpractice/Activities/WebActivity.java]: I was about to write a very complicated mechanism to download the webpage's HTML into a file in internal storage. This one-liner seems a lot better, haha.

**Comment by Gear61** [comment]
@manzhos-nikolai Submitted a new build just now to the Google Play store with these changes. Thank you for helping me with these new features! I made sure to give you credit in the app's "What's New" section.

**Comment by manzhos-nikolai** [comment]
@Gear61 Glad i can help :) Thanks for giving credits

File: app/src/main/java/randomappsinc/com/sqlpractice/Activities/MainActivity.java
Comment (wiemziadi929-gif) [inline app/src/main/java/randomappsinc/com/sqlpractice/Activities/MainActivity.java]: app/src/main/java/randomappsinc/com/sqlpractice/Activities/MainActivity.java

**Pull request merged**